smartsheet-csharp-sdk/main/Smartsheet/Api/Internal/EventResourceImpl.cs

```diff
@@ -103,27 +103,27 @@ public virtual EventResult ListEvents(object? since, string? streamPosition, int
             HttpRequest request = null;
             try
             {
-                request = CreateHttpRequest(new Uri(this.smartsheet.BaseURI, "events" + QueryUtil.GenerateUrl(null, parameters)), HttpMethod.GET);
+                request = CreateHttpRequest(new Uri(this.Smartsheet.BaseURI, "events" + QueryUtil.GenerateUrl(null, parameters)), HttpMethod.GET);
             }
             catch (Exception e)
             {
                 throw new SmartsheetException(e);
             }
 
-            HttpResponse response = this.smartsheet.HttpClient.Request(request);
+            HttpResponse response = this.Smartsheet.HttpClient.Request(request);
 
             EventResult obj = null;
             switch (response.StatusCode)
             {
                 case HttpStatusCode.OK:
-                    obj = this.smartsheet.JsonSerializer.DeserializeEventResult(response.Entity.GetContent());
+                    obj = this.Smartsheet.JsonSerializer.DeserializeEventResult(response.Entity.GetContent());
                     break;
                 default:
                     HandleError(response);
                     break;
             }
 
-            smartsheet.HttpClient.ReleaseConnection();
+            Smartsheet.HttpClient.ReleaseConnection();
```

Comment:
Previously, this `smartsheet` was referencing the local function argument. Now it is referencing the virtual property from the `AbrstractResources`. Syntactically that is the same object, but for avoidance of doubt, either change it back to lowercase `smartsheet` to reference the local, or use the explicit `this.Smartsheet` notation for consistency with the rest of the usages.

Reply:
(My local env is not working so I can't read the code in a sensible way, so maybe I'm misunderstanding it, or maybe my see-sharp is getting rusty.)

I am in agreement with both your points. The confusion for me is, we are inconsistently referencing the new `Smartsheet` property with and without the `this.` qualifier.